packages/picasso-forms/src/utils/scroll-to.ts

```diff
@@ -1,5 +1,10 @@
+const UNHIDDEN_INPUT_SELECTOR = 'input:not([type=hidden])'
```

Comment:
nit: what about this?
```suggestion
const VISIBLE_INPUT_SELECTOR = 'input:not([type=hidden])'
```

Reply:
`UNHIDDEN` is not a word. But does not hidden actually mean visible here? I guess `NOT_HIDDEN_INPUT_SELECTOR` might be better.

Reply:
Aight, @separatio will be the winner 👍